xml/Microsoft.Extensions.Logging/LoggerFactory.xml

```diff
@@ -229,7 +229,7 @@
       <Docs>
         <summary>Check if the factory has been disposed.</summary>
         <returns>
-          <see langword="true" /> when <see cref="M:Microsoft.Extensions.Logging.LoggerFactory.Dispose" /> as been called</returns>
+          <see langword="true" /> when <see cref="M:Microsoft.Extensions.Logging.LoggerFactory.Dispose" /> as been called.</returns>
```

Comment:
```suggestion
          <see langword="true" /> if <see cref="M:Microsoft.Extensions.Logging.LoggerFactory.Dispose" /> has been called; otherwise, <see langword="false" />.</returns>
```